primefaces/src/main/java/org/primefaces/component/fileupload/FileUploadBase.java

```diff
@@ -274,8 +274,7 @@ public void setOncomplete(String oncomplete) {
     }
 
     public String getOnvalidationfailure() {
-        Object eval = getStateHelper().eval(PropertyKeys.onvalidationfailure, null);
-        return (String) eval;
+        return (String) getStateHelper().eval(PropertyKeys.onvalidationfailure, null);
```

Comment:
Is it normal that `onvalidationfailure` is not camel case?

Reply:
not really :/ but not my task currently, just make the getter look like all our getters